content/Arduino Cloud/General Cloud/Wi-Fi-troubleshooting.md

```diff
@@ -0,0 +1,53 @@
+---
+title: "Wi-Fi troubleshooting"
+---
+
+This guide is intended to help you troubleshoot common Wi-Fi connectivity issues with your devices. Follow the steps below to identify and resolve potential issues.
+
+## 1. Frequency Bands: 2.4 GHz vs 5 GHz
+
+Learn more about it [here](https://support.arduino.cc/hc/en-us/articles/14144656051740-Supported-Wi-Fi-bands)
+
+## 2. Avoid common network names
+
+Avoid using generic names like "default" or "home" to prevent interference and confusion with neighboring networks.
+
+## 3. Reset Network Credentials
```

Comment:
```suggestion
## 3. Reset network credentials for your Thing
```
To emphasize that changes should be made on the Thing itself, not on the Wi-Fi network (as shown in section 2).